### PR TITLE
setPrimitivePostRequestFront

### PR DESCRIPTION
**Pull request by ilyapozh**
A very begining of the post reporting section. The conection between vue and nest is set.

### CODE REVIEW

File: my-vue3-app/src/components/SubmitReport.vue [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
     <h1>{{ msg }}</h1>
     <p>
       Сайт для наркетов!
     </p>
     <h3>Installed CLI Plugins</h3>
+  </div> -->
+  <div class="submitReportForm">
+    <h1 class="submitReportForm__title">Расскажите о ваших впечатлениях в подробностях</h1>
+      <input v-model="title" placeholder="Add title">
+      <p>The title is: {{ title }}</p>
```
Comment (tomfun) [inline my-vue3-app/src/components/SubmitReport.vue:12]: It is not necessary to see it. Let's remove it

File: my-vue3-app/src/components/SubmitReport.vue [hunk 2]
```diff
@@ -1,10 +1,25 @@
 <template>
-  <div class="hello">
+  <!-- <div class="hello">
```
Comment (tomfun) [inline my-vue3-app/src/components/SubmitReport.vue:2]: Remove it pls

File: my-vue3-app/src/components/SubmitReport.vue [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
     <h3>Installed CLI Plugins</h3>
+  </div> -->
+  <div class="submitReportForm">
+    <h1 class="submitReportForm__title">Расскажите о ваших впечатлениях в подробностях</h1>
+      <input v-model="title" placeholder="Add title">
+      <p>The title is: {{ title }}</p>
+      <br />
+      <br />
+      <span>Multiline message is:</span>
+      <p style="white-space: pre-line;">{{ reportText }}</p>
```
Comment (tomfun) [inline my-vue3-app/src/components/SubmitReport.vue:16]: rm {{ ... }}

File: my-vue3-app/src/components/SubmitReport.vue [hunk 4]
```diff
@@ -16,6 +31,37 @@ export default defineComponent({
...
+            },
+          ],
+          nick: this.reportText,
+          title: this.title,
+        }),
+      };
+      const response = await fetch('/api/report', requestOptions);
+      const data = await response.json();
+
+      console.log(data);
```
Comment (tomfun) [inline my-vue3-app/src/components/SubmitReport.vue:62]: rm

**Review by tomfun — CHANGES_REQUESTED** [review]

File: my-vue3-app/src/components/SubmitReport.vue (hunk 1 above)
Comment (ilyapozh) [reply my-vue3-app/src/components/SubmitReport.vue:12]: not necessary to see what?

Comment (tomfun) [reply my-vue3-app/src/components/SubmitReport.vue:12]: The whole line. When you enter field called 'title', you don't need to see under the field this <p>

**Pull request merged**